modules/core/include/opencv2/core/utility.hpp

```diff
@@ -264,9 +265,53 @@ execution time in seconds:
     // do something ...
     t = ((double)getTickCount() - t)/getTickFrequency();
 @endcode
+@sa getTickCount, TickMeter
  */
 CV_EXPORTS_W double getTickFrequency();
 
+/** @brief a Class to measure passing time.
+
+The class computes passing time by counting the number of ticks per second. That is, the following code computes the
+execution time in seconds:
+@code
+TickMeter tm;
+tm.start();
+// do something ...
+tm.stop();
+std::cout << tm.getTimeSec();
+@endcode
+@sa getTickCount, getTickFrequency
+*/
+
+class CV_EXPORTS_W TickMeter
+{
+public:
+    //! the default constructor
+    CV_WRAP TickMeter();
+    CV_WRAP void start();
+    CV_WRAP void stop();
+
+    CV_WRAP int64 getTimeTicks() const;
+    CV_WRAP double getTimeMicro() const;
+    CV_WRAP double getTimeMilli() const;
+    CV_WRAP double getTimeSec()   const;
+    /**
+    returns internal counter value.
+    */
+    CV_WRAP int64 getCounter() const;
+
+    /**
+    resets internal values.
+    */
+    CV_WRAP void reset();
+private:
+    int64 counter;
+    int64 sumTime;
+    int64 startTime;
+};
+
+std::ostream& operator << (std::ostream& out, const TickMeter& tm);
```

Comment:
i need some help about documentation. how can we give an example usage of << operator ?